addons/postgresql/dataprotection/postgresql-pitr-restore.sh

```diff
@@ -26,10 +26,12 @@ echo "#!/bin/bash" > ${RESTORE_SCRIPT_DIR}/kb_restore.sh;
 echo "[[ -d '${DATA_DIR}.old' ]] && mv -f ${DATA_DIR}.old/* ${DATA_DIR}/;" >> ${RESTORE_SCRIPT_DIR}/kb_restore.sh;
 echo "sync;" >> ${RESTORE_SCRIPT_DIR}/kb_restore.sh;
 chmod +x ${RESTORE_SCRIPT_DIR}/kb_restore.sh;
-echo "restore_command='case "%f" in *history) cp ${PITR_DIR}/%f %p ;; *) mv ${PITR_DIR}/%f %p ;; esac'" > ${CONF_DIR}/recovery.conf;
-echo "recovery_target_time='${DP_RESTORE_TIME}'" >> ${CONF_DIR}/recovery.conf;
-echo "recovery_target_action='promote'" >> ${CONF_DIR}/recovery.conf;
-echo "recovery_target_timeline='latest'" >> ${CONF_DIR}/recovery.conf;
+cat << EOF > "${CONF_DIR}/recovery.conf"
+restore_command='case "%f" in *history) cp ${PITR_DIR}/%f %p ;; *) mv ${PITR_DIR}/%f %p ;; esac'
+recovery_target_time='$( date -d "@${DP_RESTORE_TIMESTAMP}" '+%F %T%::z' )'
```

Comment:
PGSQL uses ISO-8601 international standard, but seems it is not a ISO-8601.  replace with` date -u -d @"${DP_RESTORE_TIMESTAMP}" +"%Y-%m-%dT%H:%M:%SZ"`?

Reply:
From the [PG documentation](https://www.postgresql.org/docs/current/runtime-config-wal.html#GUC-RECOVERY-TARGET-TIME): 
![image](https://github.com/user-attachments/assets/de2ba9b7-aeb6-40f2-be75-06b20ea1cac8)

And the timestamp format is described [here](https://www.postgresql.org/docs/current/datatype-datetime.html#DATATYPE-DATETIME-INPUT-TIME-STAMPS)
![image](https://github.com/user-attachments/assets/6a8e7736-c101-4188-90c1-f7e164deecbb)

So I just follow the PG documentation to generate the timestamp string